Examples/Algorithms/Vertexing/src/HoughVertexFinderAlgorithm.cpp

```diff
@@ -52,7 +58,7 @@ ActsExamples::SingleSeedVertexFinderAlgorithm::execute(
                                      << "mm, z = " << vtx.value()[2] << "mm");
 
     std::vector<Acts::Vertex> vertexCollection;
-    vertexCollection.emplace_back(vtx.value());
+    vertexCollection.emplace_back(Acts::Vertex({vtx.value()}));
```

Comment:
```suggestion
    vertexCollection.emplace_back({vtx.value()});
```
using `emplace_back` I would expect this to work as well?

Reply:
indeed, `emplace_back(vtx.value());` actually works! 👍